feat(changelog): load-balancers-added-host-header-routes-just 2023-03-07 (#1392)
scaleway/docs-content: changelog/march2023/2023-03-07-load-balancers-added-host-header-routes-just.mdx

---
title: Host header routes just arrived!
status: added
author:
    fullname: 'Join the #load-balancer channel on Slack.'
    url: 'https://scaleway-community.slack.com/archives/load-balancer'
date: 2023-03-07
category: network
product: load-balancers
---

Scaleway Load Balancers now provide the ability to create routes based on the value of incoming requests' Host HTTP headers.

Routes are used when a Load Balancer has multiple backends, to choose which of them to forward traffic to. There are now two types of routes:
* **SNI routes**: the only type we previously supported. They use the SNI value provided during the TLS handshake. This route type is suitable for TCP backends when TLS is enabled on the Load Balancer.

> * **Host routes**: this new route type uses the value of the Host header in the incoming HTTP request. It is perfect for HTTP backends when you need to host several services behind your Load Balancer

Reviewer: ```suggestion
* **Host routes**: this new route type uses the value of the Host header in the incoming HTTP request. It is perfect for HTTP backends when you need to host several services behind your Load Balancer.
```

> The console user interface manages everything for you and configures the route type automatically based on the frontend and backend you use. The API still lets you configure the route type but please be aware that from June 1st we won't support using SNI routes with HTTP backends, so please make sure you update your automation tools.

Reviewer: ```suggestion
The console user interface manages everything for you and configures the route type automatically based on the frontend and backend you use. The API still lets you configure the route type but make sure you update your automation tools, as from June 1st we won't support the use of SNI routes with HTTP backends anymore.
```
